libraries/bezier-rs/src/bezier/transform.rs

```diff
@@ -90,6 +90,12 @@ impl Bezier {
 		self.apply_transformation(&|point| rotation_matrix.mul_vec2(point))
 	}
 
+	/// Returns a Bezier curve that results from rotating the curve around the provided point by the given angle (in radians).
+	pub fn rotate_about_point(&self, angle: f64, origin: DVec2) -> Bezier {
```

Comment:
I always felt that the origin was the point (0, 0). How do we feel about renaming this argument `pivot`?